src/System.Text.Json/src/System/Text/Json/JsonEncodedText.cs

```diff
@@ -183,9 +183,6 @@ public override bool Equals(object obj)
         /// Converts the value of this instance to a <see cref="string"/>.
         /// </summary>
         /// <remarks>
-        /// Returns the underlying UTF-16 encoded string.
```

Comment:
I don't think we should remove this. Instead, let's make it a `returns` tag. I am open to word-smithing it differently though, if you have suggestions.

Reply:
I'm not too sure about it as well. 😅 guess I could do that.